src/ingestion/sources/redshift.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from .postgresql import PostgreSQLSource, PostgreSQLSourceConfiguration, PostgreSQLSourceDialect, PostgreSQLSourceExtractor
+
+class RedshiftSourceConfiguration(PostgreSQLSourceConfiguration):
+    def _connection_string_prefix(self) -> str:
+        return "redshift+psycopg2"
+
+    def connection_string(self) -> str:
+        return super().connection_string()
+
+    @property
+    def type(self):
+        return "redshift"
+
+class RedshiftSourceDialect(PostgreSQLSourceDialect):
```

Comment:
can theoretically comment this entire class to show that we aren't using it at the moment

Reply:
commented out